sunpy/util/config.py

```diff
@@ -1,11 +1,23 @@
 """SunPy configuration file functionality"""
 import os
-import tempfile
+from pathlib import Path
 import configparser
 
 import sunpy
+from sunpy.extern.appdirs import AppDirs
 
-__all__ = ['load_config', 'print_config']
+
+__all__ = ['load_config', 'print_config', 'CONFIG_DIR']
+
+# This is to avoid creating a new config dir for each new dev version.
+# We use AppDirs to locate and create the config directory.
+dirs = AppDirs("sunpy", "sunpy")
+# Default one set by AppDirs
+CONFIG_DIR = dirs.user_config_dir
+
+# This is to avoid creating a new config dir for each new dev version.
+# We use AppDirs to locate and create the config directory.
+dirs = AppDirs("sunpy", "sunpy")
```

Comment:
Why is this repeated also on l14?

Reply:
Because I can't do a rebase merge correctly.